http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/ProxyRoutes.scala

```diff
@@ -32,92 +32,94 @@ class ProxyRoutes(proxyService: ProxyService, corsSupport: CorsSupport, refererC
 ) extends LazyLogging {
   val route: Route =
     traceRequestForService(serviceData) { span =>
-      extractAppContext(Some(span)) { implicit ctx =>
-        // Note that the "notebooks" path prefix is deprecated
-        pathPrefix("proxy" | "notebooks") {
+      extractRequest { request =>
```

Comment:
diff looks big...But really all did is adding this outer ` extractRequest { request =>`